src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -9387,6 +9387,49 @@ def test_aks_create_dualstack_with_default_network(self, resource_group, resourc
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+    
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_update_dualstack(self, resource_group, resource_group_location):
```

Comment:
Please commit the corresponding recording file (find it from pipeline artifact if the live test passed).

Reply:
thanks, updated